### PR TITLE
Custom ucr databases

### PR DESCRIPTION
**Pull request by TylerSheffels**
This adds the ability to specify a custom database that will be created in postgres

### CODE REVIEW

**Comment by TylerSheffels** [comment]
@benrudolph, also corresponds with: https://github.com/dimagi/commcare-hq/pull/8568

**Comment by benrudolph** [comment]
solid

File: ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 1]
```diff
@@ -29,6 +29,12 @@ SQL_REPORTING_OBJECT_OWNER = '{{ localsettings.REPORT_OBJECT_OWNER }}'
 UCR_DATABASE_URL = "postgresql://{{ localsettings.PG_DATABASE_USER }}:{{ localsettings.PG_DATABASE_PASSWORD }}@{{ localsettings.PG_DATABASE_HOST }}/{{ localsettings.UCR_DATABASE_NAME }}"
 {% endif %}
 
+CUSTOM_DATABASES {
+    {% for custom_database in CUSTOM_DATABASES %}
+    {{ custom_database }}: {{ custom_database | upper }}_DATABASE_URL = "postgresql://{{ localsettings.PG_DATABASE_USER }}:{{ localsettings.PG_DATABASE_PASSWORD }}@{{ localsettings.PG_DATABASE_HOST }}/{{ custom_database }}"
```
Comment (benrudolph) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: is this valid python? trying to recreate in my head:
```
{
    custom_ucr_database: CUSTOM_UCR_DATABASE_DATABASE_URL = "postgresql://user:pass@host/custom_ucr_database
}
```

**Comment by benrudolph** [comment]
mgd, but have one potentially blocking question. don't think anyone is gonna deploy postgres over the weekend so not going to bother reverting

**Comment by benrudolph** [comment]
ah i'll revert just to be safe, sorry about that

File: ansible/roles/commcarehq/templates/localsettings.py.j2 (hunk 1 above)
Comment (TylerSheffels) [inline ansible/roles/commcarehq/templates/localsettings.py.j2]: Doesn't look like its even close to valid python... Not sure how I managed that.

**Pull request merged**